DemoApp/controller/user.controller.js

```diff
@@ -59,14 +49,13 @@ const userRegister=async(req,res,employeetype)=>{
     console.log("Saved")  
     await NewUser.save();
     const file_upload = await new Report({
-      // filepath: filepath,
       u_id: NewUser._id,
     }).save();
-    // return successResponse(req,res,NewUser,SUCCESSFULLY_REGISTERED,200)
+
     return res.redirect('/login');
```

Comment:
Redirection should be done from front end and you should only send a flag OR URL.

Reply:
But here we are using ejs templating module to integrate front end. In that case can we also do the same as you said ??

Reply:
Oh, Ok, if you're using EJS then it's fine.

Reply:
No need to change anything then, redirect will work. no issue.